Cesium3DTilesSelection/include/Cesium3DTilesSelection/TilesetContentLoader.h

```diff
@@ -108,6 +109,25 @@ struct CESIUM3DTILESSELECTION_API TileChildrenResult {
   TileLoadResultState state;
 };
 
+/**
+ * @brief An interface to query heights from a tileset that can do so
+ * efficiently without necessarily downloading individual tiles.
+ */
+class CESIUM3DTILESSELECTION_API ITilesetHeightQuery {
```

Comment:
Definitely a nitpick but I think `EllipsoidTilesetLoader` makes more sense as a "height sampler", or some other verbiage, that as a "height query" -- since the latter seems like the _result_ returned by a height sampler.

```suggestion
class CESIUM3DTILESSELECTION_API ITilesetHeightSampler {
```

Reply:
I'm not attached to the name though :smile: But for me, `getHeightSampler` reads more intuitively as "something that I can query heights on" rather than `getHeightQuery`

Reply:
Good point, especially since we already have a (private) `TilesetHeightQuery`. I've renamed the interface to `ITilesetHeightSampler`, renamed the method to `sampleHeights`, and moved it all to its own header file.

Reply:
The changes look great, thanks!